journey/app/controllers/profiles_controller.rb

```diff
@@ -43,22 +55,35 @@ def update_user_if_needed
     return if userparams.blank?
 
     current_user.update!(user_params)
-    set_prefs_cookies_from_params(userparams)
+    prefs_cookies_from_params(userparams)
```

Comment:
Please use proper naming conventions
```suggestion
    prefs_cookies_from_params(user_params)
```